extensions/extended-blocks/core-embed/instagram.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { addFilter } from '@wordpress/hooks';
+import { registerBlockVariation, unregisterBlockVariation } from '@wordpress/blocks';
+import { G, Path, SVG } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const embedInstagramIcon = (
+	<SVG viewBox="0 0 24 24">
```

Comment:
I think it would be nice to move those to `extensions/shared/icons.js`, with the rest of our icons.

Reply:
Right, Facebook is actually already there. I thought that maybe they looked a bit different, but the only visible difference (at that scale) seems to be blue vs black:

Before:
![image](https://user-images.githubusercontent.com/96308/94557582-0e4ce080-025f-11eb-93bc-3015b65ad8a1.png)


After:
![image](https://user-images.githubusercontent.com/96308/94557276-a39ba500-025e-11eb-893f-4809261ba275.png)

I guess the de-duplication outweighs the color change. Maybe we can even modify the component to allow it to inject additional props/colors later; or leave as-is to emphasise that this is different from the Core-provided embed.

I'll change :+1:

Reply:
Ah, interesting. When moving the Instagram icon, I noticed that we're already using one for the Lastest Insta Posts block:

![image](https://user-images.githubusercontent.com/96308/94559462-92a06300-0261-11eb-91c1-b5f18ba9138f.png)

I tracked it down to this code: https://github.com/Automattic/jetpack/blob/1527e0be4d25312d5fb4672a3f856e11ad6d84db/extensions/blocks/instagram-gallery/index.js#L21-L24

Turns out these are coming from [dashicons](https://developer.wordpress.org/resource/dashicons/#media-code), which have both an FB and an IG icon.

I'll try using those, and setting their color to JP green (to emphasize that they're coming from JP).

Reply:
d3df472:

![image](https://user-images.githubusercontent.com/96308/94559657-dabf8580-0261-11eb-989f-23fc7c0967c7.png)

![image](https://user-images.githubusercontent.com/96308/94559680-e448ed80-0261-11eb-9ff0-225c2bd286d9.png)